src/rshim.c

```diff
@@ -213,6 +250,10 @@ static uint32_t rshim_timer_interval = RSHIM_TIMER_INTERVAL;
 
 static void rshim_fifo_msg_update_checksum(rshim_tmfifo_msg_hdr_t *hdr);
 
+static int rshim_update_locked_mode(rshim_backend_t *bd);
+
+static int handle_ownership_transfer(rshim_backend_t *bd);
```

Comment:
rshim_handle_ownership_transfer()? So the naming is more consistent except for very short utility functions.